synapse/federation/transaction_queue.py

```diff
@@ -69,8 +69,10 @@ def __init__(self, hs):
         # done
         self.pending_transactions = {}
 
-        metrics.register_callback(
-            "pending_destinations",
+        LaterGauge(
+            "synapse_federation_client_pending_destinations",
```

Comment:
We're in `synapse_federation_transaction_queue_`

Reply:
Fixed.